### PR TITLE
chore(launch): new shard for launch tests

### PR DESCRIPTION
**Pull request by gtarpenning**
Jira Ticket:
https://wandb.atlassian.net/browse/WB-11400

Description
-----------
Adds to a new circle test for specific launch related tests in the new relay-server suite.

Testing
-------
Testing in draft PR, circle CI check

### CODE REVIEW

**Comment by gtarpenning** [comment]
@kptkin Do you think these tests aren't passing because of something related to the Circle changes in this PR?

**Comment by kptkin** [comment]
> @kptkin Do you think these tests aren't passing because of something related to the Circle changes in this PR?

probably, will need to debug the config setup

File: pytest.ini [hunk 1]
```diff
@@ -18,4 +18,5 @@ norecursedirs =
     tests/functional_tests
     tests/standalone_tests
     tests/unit_tests
-    tests/unit_tests_old/tests_*
+    tests/unit_tests/tests_*
+    tests/unit_tests_old/tests_*
```
Comment (kptkin) [inline pytest.ini:22]: mind fixing new line?

**Review by kptkin — APPROVED** [review]

**Review by dmitryduev — APPROVED** [review]
🎉

**Pull request merged**